vcs/VTKPlots.py

```diff
@@ -37,7 +37,6 @@ def __init__(self, parent):
         self.AddObserver(
             "LeftButtonReleaseEvent",
             parent.leftButtonReleaseEvent)
-        self.AddObserver("ModifiedEvent", parent.configureEvent)
```

Comment:
@danlipsa did you try it on MAc as well (I'm about to anyway). I remember WAY back then that there were some issues with Mac not being able to differentiate between ModifiedEvent and ConfigureEvent or something like that.

Reply:
Yes, I did tried it on mac, in vcs-js context.

Reply:
It works nicely. Using plane vcs, I get a window without a frame that I cannot resize. Is this a rencent occurence?